dashboard/app/models/script.rb

```diff
@@ -1148,50 +1148,63 @@ def fix_lesson_positions
     end
   end
 
-  def clone_migrated_unit(new_name, new_level_suffix: nil, destination_unit_group_name: nil, version_year: nil, family_name:  nil)
+  def clone_migrated_unit(new_name, new_level_suffix: nil, destination_unit_group_name: nil, destination_professional_learning_course: nil, version_year: nil, family_name:  nil)
     raise 'Script name has already been taken' if Script.find_by_name(new_name) || File.exist?(Script.script_json_filepath(new_name))
 
+    if destination_professional_learning_course.nil? && old_professional_learning_course?
```

Comment:
Do we want this check in the lesson copy as well? Said another way, should curriculum writers be able to copy a lesson from a non-DLP course to a DLP course? (The answer might be yes, so I'm just checking)

Reply:
I added an error to lesson copy to not allow copying DLP lessons to be copied to non-DLP lessons. I think its ok if something non-DLP gets copied to DLP although it would probably lose the vocab and resources associated with it. I also added a backlog item to decide if we care enough to make lesson copy work in the UI for DLP courses https://codedotorg.atlassian.net/browse/PLAT-1802.